nasa-gibs/worldview
# WV-2865 Redux 5.x Update

## Description
This updates Redux to Redux 5.0.1, as well as updating related packages and adding Redux Toolkit.

## How To Test
1. `git checkout wv-2972-redux-5.0`
2. `npm ci`
3. `npm run watch`
4. Verify that no behavior is changed since before this update.
